docs/user/dql/functions.rst

```diff
@@ -2730,6 +2730,99 @@ Another example to show how to set custom values for the optional parameters::
     +------------+
 
 
+MATCHQUERY
```

Comment:
if matchquery and match_query are synonyms of match. could we simply the doc. e.g. https://github.com/Bit-Quill/opensearch-project-sql/blob/integ-add-legacy-syntax-for-match-function/docs/user/dql/functions.rst#curdate